tests/components/selection_input.test.ts

```diff
@@ -221,24 +220,29 @@ describe("Selection Input", () => {
     expect(model.getters.getSelectionInput(id)[0].xc).toBe("C2");
   });
 
-  test("manually input multiple cells", async () => {
-    const { model, id } = await createSelectionInput();
-    await writeInput(0, "C2,A1");
-    expect(fixture.querySelectorAll("input")[0].value).toBe("C2");
-    expect(model.getters.getSelectionInput(id)[0].xc).toBe("C2");
-    expect(fixture.querySelectorAll("input")[1].value).toBe("A1");
-    expect(model.getters.getSelectionInput(id)[1].xc).toBe("A1");
-  });
-
-  test("manually add another cell", async () => {
+  test("manually add another range via trailing comma", async () => {
     const { model, id } = await createSelectionInput({ initialRanges: ["C2"] });
-    await writeInput(0, "C2,A1");
+    await writeInput(0, "C2,");
```

Comment:
We should keep the test `C2,A1`, this is still possible for it to happen if the user were to paste the value "C2,A1" directly in the input